libraries/AP_Scripting/AP_Scripting.h

```diff
@@ -56,6 +60,27 @@ class AP_Scripting
     };
     uint16_t get_disabled_dir() { return uint16_t(_dir_disable.get());}
 
+    struct mavlink_msg {
+        mavlink_message_t msg;
+        mavlink_channel_t chan;
+    };
+
+    struct mavlink_output {
+        char data[256]; // maximum payload size
```

Comment:
this needs to be max size including headers etc, should use defines from generated mavlink. Can certainly be more than 256

Reply:
we are using full mavlink_message_t struct to record the message.